packages/bierzo-wallet/src/routes/payment/index.tsx

```diff
@@ -90,6 +93,8 @@ const Payment = (): JSX.Element => {
       console.error(error);
       toast.show("An error ocurred", ToastVariant.ERROR);
       return;
+    } finally {
```

Comment:
I think the `return;` must be removed in order to close this in the exception case